.github/test.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import subprocess
+import sys
+import tuttest
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--first', action='store_true')
```

Comment:
`--first` ? Can we name it in a way it's easy to figure out what it does?

Reply:
How about `--first-run`? See new commit.